Cargo.lock

```diff
@@ -1,2001 +1,2001 @@
+# This file is automatically @generated by Cargo.
```

Comment:
It seems to be this doesn't update any dependencies, but only format of `Cargo.lock` file. I guess https://github.com/kinvolk/afterburn/blob/flatcar-master/Cargo.toml would have to be modified to update the dependencies.

Reply:
Depends on whether the versions are specified exactly in `Cargo.toml` this here can pull in updates. I didn't look if that's the case but anyway we need to make changes in the ebuild file if we want to update some dependency version. Also it's always good to test if the ebuild still works when you do changes in a project and bump the ebuild to the latest commit, or it becomes a trap for the next person woring on it.